srml/system/src/lib.rs

```diff
@@ -314,7 +322,10 @@ decl_storage! {
 		/// Total extrinsics count for the current block.
 		ExtrinsicCount: Option<u32>;
 		/// Total weight for all extrinsics put together, for the current block.
-		AllExtrinsicsWeight: Option<u32>;
+		AllExtrinsicsWeight: Option<Weight>;
+		/// The next fee multiplier. This should be updated at the end of each block based on the
+		/// saturation level (weight).
+		NextFeeMultiplier: FeeMultiplier = Default::default();
```

Comment:
"fees" are not a subject that this module should be aware of. if it's a weight multiplier, then rename. if it's actually a fee multiplier (and operates on e.g. balances), then move to either balances module or its own "fees" module.